qiskit/pulse/__init__.py

```diff
@@ -145,9 +135,251 @@
 
    PulseError
 
-"""
+Pulse Builder (:mod:`~qiskit.pulse.builder`)
+===================================================
+
+The pulse builder provides an imperative API for writing pulse programs
+with less difficulty than the :class:`~qiskit.pulse.Schedule` API.
+It contextually constructs a pulse schedule and then emits the schedule for
+execution. For example to play a series of pulses on channels is as simple as:
+
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.visualization import SchedStyle
+
+    dc = pulse.DriveChannel
+    d0, d1, d2, d3, d4 = dc(0), dc(1), dc(2), dc(3), dc(4)
+
+    with pulse.build(name='Pulse Programming in') as pulse_prog:
+        pulse.play([1, 1, 1, 0, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 1, 1, 0, 1, 1, 1], d0)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0], d1)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0], d2)
+        pulse.play([1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0], d3)
+        pulse.play([1, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 1, 0, 1, 1, 1, 0, 0, 1, 0], d4)
+
+    style = SchedStyle(figsize=(8, 4))
+    pulse_prog.draw(style=style)
+
+
+In the future the pulse builder will be coupled to the
+:class:`~qiskit.circuit.QuantumCircuit` with an equivalent circuit builder
+interface.
+
+.. warning::
+    The pulse builder interface is still in active development. It may have
+    breaking API changes without deprecation warnings in future releases until
+    otherwise indicated.
+
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.build
+
+Channels
+--------
+Methods to return the correct channels for the respective qubit indices.
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.test.mock import FakeArmonk
+
+    backend = FakeArmonk()
+
+    with pulse.build(backend) as drive_sched:
+        d0 = pulse.drive_channel(0)
+        print(d0)
+
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.acquire_channel
+    ~qiskit.pulse.builder.control_channels
+    ~qiskit.pulse.builder.drive_channel
+    ~qiskit.pulse.builder.measure_channel
+
+Instructions
+------------
+Pulse instructions available within the builder interface.
```

Comment:
Maybe something like this?
```suggestion
Pulse instructions are available within the builder interface. Here's an example:
```